test/matrix/dense_kernels.cpp

```diff
@@ -135,17 +121,23 @@ class Dense : public ::testing::Test {
         dc_y = gko::clone(exec, c_y);
         dalpha = gko::clone(exec, alpha);
         dc_alpha = gko::clone(exec, c_alpha);
+        result = Mtx::create(ref, gko::dim<2>{1, num_vecs});
+        dresult = Mtx::create(exec, gko::dim<2>{1, num_vecs});
     }
 
     void set_up_apply_data()
     {
         x = gen_mtx<Mtx>(40, 25);
```

Comment:
That's the size from the OMP kernels. I'd suggest taking the bigger size from the other executors to include at least two warps everywhere